app.rb

```diff
@@ -1,163 +1,20 @@
-require_relative './classroom'
+require_relative './book'
+require_relative './person'
 require_relative './student'
 require_relative './teacher'
-require_relative './book'
 require_relative './rental'
+require_relative './modules/show_books'
+require_relative './modules/show_person'
+require_relative './modules/create_person'
+require_relative './modules/create_book'
+require_relative './modules/create_rental'
+require_relative './modules/list_rental_id'
```

Comment:
- [ ] I think you have the wrong path here since your module called list_rental but you are calling it as list_rental_id so I think you should update the name path here so please fix it 🙏🏻💯🔥



![image](https://user-images.githubusercontent.com/58553711/211436655-b82ccca2-d4b9-45a5-bdc9-47e55768b301.png)